### PR TITLE
added tests to the project

### PR DESCRIPTION
**Pull request by AnyaRatusnyak**

### CODE REVIEW

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@AnyaRatusnyak Hi) in general good job... but why you choose so simple class to teses? I would ask you too add more tests.. just for practice)) and in task we have to use biggest class :) 
image

**Comment by AnyaRatusnyak** [comment]
Oh, sorry ! It was a misunderstanding! I thought that I needed to make tests for the class from Spring tasks

File: src/test/java/ticketsproject/TicketServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    @Test
+    void findById_ExistingId_ReturnsTicket() {
+        Ticket ticket = TicketService.findById(ID, tickets);
+        assertNotNull(ticket);
+        assertEquals(ID, ticket.getId());
+    }
+
+    @Test
+    void findById_NonExistingId_ThrowsEntityNotFoundException() {
+        NoSuchElementException exception = assertThrows(NoSuchElementException.class, () -> {
```
Comment (Alexmansar) [inline src/test/java/ticketsproject/TicketServiceTest.java]: method name has ThrowsEntityNotFoundException, but in method we check NoSuchElementException
its correct?

**Review by Alexmansar — APPROVED** [review]
@AnyaRatusnyak Hi)) thanks for m=new tests)
I have 1 small comment. plese fix/comment it, and merge))

File: src/test/java/ticketsproject/TicketServiceTest.java (hunk 1 above)
Comment (AnyaRatusnyak) [reply src/test/java/ticketsproject/TicketServiceTest.java]: My bad!! Thanks a million !! Have a nice day !!!

**Pull request merged**